card/config/initializers/02_patches/kaminari.rb

```diff
@@ -12,6 +12,7 @@ def page_url_for page
           p = params_for(page)
           p.delete :controller
           p.delete :action
+          # TODO: make sure paging works with new related handling
```

Comment:
I still need to check this, but since everything is inside a slot now, I think I can actually delete the following lines.